deploy: v1.0.0

ho991217 opened the pull request.

ho991217 reviewed (approved):
LGTM^^

DoctorK4 reviewed (approved):
확인했습니다! 
vscode 익스텐션 설정은 차차 같이 알아보도록 하겠습니다.

ho991217 committed:
Create SECURITY.md

Signed-off-by: 이호연 <ho991217@kakao.com>

dependabot[bot] committed:
build(deps): bump next from 14.1.0 to 14.1.1

Bumps [next](https://github.com/vercel/next.js) from 14.1.0 to 14.1.1.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v14.1.0...v14.1.1)

---
updated-dependencies:
- dependency-name: next
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

ho991217 committed:
Merge pull request #2 from ToyVallet/dependabot/npm_and_yarn/next-14.1.1

build(deps): bump next from 14.1.0 to 14.1.1

ho991217 committed:
feat: eslint 세팅

kd02109 committed:
dep: yarn, 라이브러리 설치

kd02109 committed:
dep: yarn 패키지 설치

kd02109 committed:
feat: custom fetch server 동작시 cookie 삽입 추가

ho991217 committed:
chore: prettierrc 추가

kd02109 committed:
feat : user 조회 관련 api 연결 및 type 생성

ho991217 committed:
feat: typescript config 추가

kd02109 committed:
feat: 사용자 다건 조회 api 연결

ho991217 committed:
feat: tsconfig 수정

ho991217 committed:
chore: husky 세팅

kd02109 committed:
fix: api 수정

kd02109 committed:
feat: user table 생성

kd02109 committed:
fix: pagenation table prop으로 수정

ho991217 committed:
feat: 절대 경로로 변경

kd02109 committed:
feat: 사용하지 않는 기능 제거

ho991217 committed:
chore: cn 함수 공통 분리

ho991217 committed:
chore: device 유틸 함수 공통 분리

ho991217 committed:
chore: 테스트 세팅

ho991217 committed:
chore: cleanup

ho991217 committed:
Merge pull request #6 from ToyVallet/chore/54-migrate-to-monorepo

[TASK-54] migrate to monorepo

ho991217 committed:
feat: storybook 추가

kd02109 committed:
dep: yarn 패키지 설치

kd02109 committed:
Merge branch 'dev' into feature-rootadmin/28-user-page

ho991217 committed:
feat: tailwindcss 세팅

kd02109 committed:
fix: conflict 해결

ho991217 committed:
Merge pull request #5 from ToyVallet/feature-rootadmin/28-user-page

[TASK-28]루트 어드민 유저 페이지 기능

ho991217 committed:
chore: tailwind and stroybook

ho991217 committed:
Merge remote-tracking branch 'origin/dev' into chore/54-migrate-to-monorepo

ho991217 committed:
chore: lint warning resolve

ho991217 committed:
chore: 스토리북 삭제

ho991217 committed:
chore: lint-staged 세팅

ho991217 committed:
chore(@swifty/ui): tailwind 세팅

ho991217 committed:
chore: 공용 tailwind config 사용

ho991217 committed:
chore: shadcn 세팅

ho991217 committed:
chore: 버튼 세팅

ho991217 committed:
chore: intl 제거

ho991217 committed:
chore: storybook ci

ho991217 committed:
fix(@swifty/shared-lib): jest가 구동되지 않는 문제 해결

ho991217 committed:
chore(@swifty/ui): button disabled 상태 수정

ho991217 committed:
Merge pull request #7 from ToyVallet/chore/54-migrate-to-monorepo

[TASK-54] chore(@swifty/ui): 스토리북 세팅

ho991217 committed:
Create deploy-storybook-github-pages.yml

Signed-off-by: 이호연 <ho991217@kakao.com>

ho991217 committed:
chore: yarn 버전 다운그레이드

hyunwleeee committed:
mv: table-board to concert-board

hyunwleeee committed:
feat: getAllFestival fetch api

hyunwleeee committed:
Merge branch 'chore/54-migrate-to-monorepo' of https://github.com/ToyVallet/swifty-frontend into feature(root-admin)/131-festival-page

kd02109 committed:
Merge pull request #8 from ToyVallet/chore/54-migrate-to-monorepo

chore: yarn 버전 다운그레이드

kd02109 committed:
feat: 쿠키 삽입 함수 생성

kd02109 committed:
feat: user table tag 디자인 추가

kd02109 committed:
feat : table css 수정

kd02109 committed:
feat : breadcrumb 구현

kd02109 committed:
fix: 철자 수정

kd02109 committed:
feat: 유저 상세 페이지 작성

kd02109 committed:
refactor: user action btton 기능 분리

kd02109 committed:
fix: breadcrumb 이름 수정

kd02109 committed:
refactor: UserActions 리펙토링

kd02109 committed:
fix: server 요청 revalidation 적용

kd02109 committed:
Merge pull request #9 from ToyVallet/feature-rootadmin/28-user-page

[TASK-13]<root-admin>루트 어드민 유저 상태 변경 기능 추가

kd02109 committed:
fix: 버튼 리스트 낙관적 업데이트 로직 수정

hyunwleeee committed:
feat: concert 화면, API 연동

hyunwleeee committed:
feat: festival 화면, GET(total, detail) API 연동

hyunwleeee committed:
feat: card 컴포넌트 구현

hyunwleeee committed:
feat: drawer-button 컴포넌트 구현

hyunwleeee committed:
feat: lineup 화면, API 연동 중

hyunwleeee committed:
Merge branch 'chore/54-migrate-to-monorepo' of https://github.com/ToyVallet/swifty-frontend into feature(root-admin)/131-festival-page

ho991217 committed:
chore: customFetch를 서버에서도 쿠키를 담아보낼 수 있도록 변경

ho991217 committed:
chore: src 라우트로 변경

ho991217 committed:
feat: if 컴포넌트 추가

ho991217 committed:
feat: choose-when 컴포넌트 추가

kd02109 committed:
dep: tailwindcss

ho991217 committed:
feat: choose-when 컴포넌트 적용

ho991217 committed:
feat: 루트 페이지 수정

ho991217 committed:
feat: 상세페이지 네비게이션 수정

ho991217 committed:
feat: 고장난 부분 모두 수정

ho991217 committed:
chore: 축제 리팩토링

ho991217 committed:
feat: 캐러셀 간격 수정

ho991217 committed:
chore: 캐러셀 인디케이터 스타일 수정

ho991217 committed:
chore: 캐러셀 인디케이터 두자리로 수정

ho991217 committed:
Update packages/shared-lib/src/utils/server/get-serverside-cookies.ts

Co-authored-by: kd02109 <57277708+kd02109@users.noreply.github.com>
Signed-off-by: 이호연 <ho991217@kakao.com>

ho991217 committed:
chore: 버튼 컴포넌트 수정

ho991217 committed:
docs: 버튼 outlined 스토리 추가

ho991217 committed:
chore: 버튼 컴포넌트 완전 이전

ho991217 committed:
fix: framer-motion 수정

kd02109 committed:
fix: User Status UI 수정

ho991217 committed:
Merge pull request #11 from ToyVallet/feature/181-refactor-customFetch

[TASK-181] chore(@swifty/shared-lib): customFetch를 서버에서도 쿠키를 담아보낼 수 있도록 변경

ho991217 committed:
chore: added "Chromatic Deployment" workflow

ho991217 committed:
chore: 노드 버전 변경

ho991217 committed:
Merge branch 'chore/199-chromatic-ci' of github.com:ToyVallet/swifty-frontend into chore/199-chromatic-ci

ho991217 committed:
chore: 크로마틱 배포

ho991217 committed:
delet

ho991217 committed:
fe

ho991217 committed:
fix: 스토리북 동작하지 않는 문제 해결

ho991217 committed:
chore: node version to v18

kd02109 committed:
Merge branch 'dev' into feature(root-admin)/183-button-list-ui

ho991217 committed:
build-storybook으로 변경

ho991217 committed:
chore: updated ci

ho991217 committed:
chore: 크로마틱 세팅

ho991217 committed:
chore: 테스트를 위한 스토리북 변경

ho991217 committed:
Merge branch 'dev' into fix(user-web)/187-fix-user-web

ho991217 committed:
Merge pull request #15 from ToyVallet/fix(user-web)/187-fix-user-web

[TASK-187] fix(user web): 유저 웹 수정

ho991217 committed:
Merge branch 'dev' into chore/199-chromatic-ci

dependabot[bot] committed:
build(deps): bump jsonwebtoken from 8.5.1 to 9.0.0

Bumps [jsonwebtoken](https://github.com/auth0/node-jsonwebtoken) from 8.5.1 to 9.0.0.
- [Changelog](https://github.com/auth0/node-jsonwebtoken/blob/master/CHANGELOG.md)
- [Commits](https://github.com/auth0/node-jsonwebtoken/compare/v8.5.1...v9.0.0)

---
updated-dependencies:
- dependency-name: jsonwebtoken
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

ho991217 committed:
chore: packages/ui, stories/**, .storybook/** 이  변경되었을 때만 스토리북 배포하도록 수정

ho991217 committed:
chore: 크로마틱 배포

ho991217 committed:
Merge branch 'dev' into chore/199-chromatic-ci

ho991217 committed:
Merge pull request #17 from ToyVallet/chore/199-chromatic-ci

[TASK-190] chore: 크로마틱 CI 작업

ho991217 committed:
Merge pull request #1 from ToyVallet/dependabot/npm_and_yarn/jsonwebtoken-9.0.0

build(deps): bump jsonwebtoken from 8.5.1 to 9.0.0

ho991217 committed:
chore: 푸시 할 때 브랜치 한정

kd02109 committed:
Merge branch 'dev' into feature(root-admin)/183-button-list-ui

kd02109 committed:
feat : root-admin 파일 위치 재조정

kd02109 committed:
fix: children이 두번 정의 되어 있는 문제 수정

hyunwleeee committed:
chore: remove moment, add dayjs

hyunwleeee committed:
fix: datePicker 날짜 오류 수정

hyunwleeee committed:
mv: card to panel

hyunwleeee committed:
feat: open, hidden concert api (optimistic 업데이트)

ho991217 committed:
Update README.md

Signed-off-by: 이호연 <ho991217@kakao.com>

kd02109 committed:
feat: University Table 생성

kd02109 committed:
feat: 대학교 정보 수정 작성

kd02109 committed:
feat: 대학 생성, 대학 로고 수정

kd02109 committed:
fix: 유저 pause 상태 버튼에 반영 안되는 버그 수정

kd02109 committed:
fix: 대학 생성 버튼 페이지 수정

ho991217 committed:
Merge pull request #18 from ToyVallet/feature(root-admin)/183-button-list-ui

[TASK-183] feaure(root-admin): 루트 어드민 유저 상세 페이지 UI 조정

hyunwleeee committed:
feat: concert api 요청

hyunwleeee committed:
feat: lineup api 요청

kd02109 committed:
feat: 대학교 검색 API 연결

kd02109 committed:
fix: 대학 테이블 column 로고 주소 정보 추가

kd02109 committed:
feat: 배럴 파일 도입

kd02109 committed:
fix: 대학 생성 logo 로직 수정

hyunwleeee committed:
Merge branch 'dev' of https://github.com/ToyVallet/swifty-frontend into feature(root-admin)/131-festival-page

hyunwleeee committed:
feat: 임시 코드 in festival

1. 급한대로 multipart-form 전송하면서 custom-fetch 함수 헤더의 application/json을
   덮어쓰기를 하지 못해 주석처리하고 사용했습니다. multipart-form에
   관해서 migration 해주시면 좋을 것 같습니다.
2. 낙관적 없데이트 결국 마무리 못 지었습니다. 콘서트 open, hidden처리
   미완성
3. 라인업 open, hidden 처리 아직 못했습니다. (백엔드 api TASK-194
   기다리는 중)
4. 라인업 update 처리 아직 못했습니다. (백엔드 api에서 이전 이미지가 null일 경우 수정이 안되는 부분 수정
   기다리는 중)

kd02109 committed:
Merge branch 'dev' into feature(root-admin)/150-university

kd02109 committed:
feat: University detail page breadcrumb

kd02109 committed:
feat: 대학 상세 페이지 호스트 유저 조회 기능

kd02109 committed:
feat: 대학 상세 페이지 축제 보기

kd02109 committed:
feat: 대학 축제 생성 기능 추가

kd02109 committed:
refactor: delte button 컴포넌트 구분

kd02109 committed:
Merge branch 'feature(root-admin)/199-detail-university-festivals' into feature(root-admin)/131-festival-page

kd02109 committed:
refactor: 불필요한 코드 제거

kd02109 committed:
fix: 콘서트 업데이트 수정

kd02109 committed:
fix: 콘서트 생성 로직 수정

kd02109 committed:
fix: 라인업 생성 경로 수정

kd02109 committed:
fix: 라인업 수정 로직 수정

kd02109 committed:
feat: 라인업 삭제

kd02109 committed:
feat: 축제, 콘서트, 라인업 상태 변경 로직 추가

dependabot[bot] committed:
build(deps): bump ws from 7.5.9 to 7.5.10

Bumps [ws](https://github.com/websockets/ws) from 7.5.9 to 7.5.10.
- [Release notes](https://github.com/websockets/ws/releases)
- [Commits](https://github.com/websockets/ws/compare/7.5.9...7.5.10)

---
updated-dependencies:
- dependency-name: ws
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

kd02109 committed:
feat: 콘서트 페이지 콘서트 수정, 삭제, 라인업 추가 버튼 생성

kd02109 committed:
feat : Festival 수정 삭제 기능 추가

kd02109 committed:
fix: 대학 축제 수정 이미지 처리 로직 수정

kd02109 committed:
fix: form 데이터 초기화 로직 추가

kd02109 committed:
feat: 콘서트 페이지 페스티벌 description 추가

kd02109 committed:
feat: global error page 작성

kd02109 committed:
feat: error.tsx 스타일 및 파일명 수정

kd02109 committed:
chore: build 과정시 발생하는 error 수정

kd02109 committed:
feat: antd notification 적용

ho991217 committed:
Merge pull request #10 from ToyVallet/feature(root-admin)/131-festival-page

[TASK-131] feaure(root-admin): 루트 어드민 festival 작업

ho991217 committed:
Merge branch 'dev' into dependabot/npm_and_yarn/ws-7.5.10

Signed-off-by: 이호연 <ho991217@kakao.com>

ho991217 committed:
Merge pull request #22 from ToyVallet/dependabot/npm_and_yarn/ws-7.5.10

build(deps): bump ws from 7.5.9 to 7.5.10

kd02109 committed:
feat: 로그인 페이지 적용

kd02109 committed:
feat: user 페이지 notification 적용

kd02109 committed:
feat : university페이지 notification 처리

kd02109 committed:
feat: festivals notification 추가

kd02109 committed:
Merge branch 'dev' into featuer(root-admin)/136-nextjs-error

kd02109 committed:
remove: 필요없는 파일 삭제

kd02109 committed:
fix: handleNotification의 type을 선택적 인자로 변경

ho991217 committed:
Merge branch 'dev' into feature(root-admin)/128-admin-notification

kd02109 committed:
fix: import 수정

kd02109 committed:
Merge pull request #25 from ToyVallet/featuer(root-admin)/136-nextjs-error

[TASK-136]루트 어드민 error.tsx 추가

kd02109 committed:
Merge branch 'dev' into feature(root-admin)/128-admin-notification

Signed-off-by: kd02109 <57277708+kd02109@users.noreply.github.com>

kd02109 committed:
Merge pull request #26 from ToyVallet/feature(root-admin)/128-admin-notification

[TASK-128] 루트 어드민 알림 기능

ho991217 committed:
hotfix: 빌드 오류 해결

ho991217 committed:
hotfix: 빌드 오류 해결

ho991217 committed:
hotfix: 빌드 오류 해결

kd02109 reviewed (approved):
LGTM
